index.html

```diff
@@ -190,8 +190,28 @@ <h2 class="section-title">SCHEDULE</h2>
                 <h3> Registration </h3>
               </li>
               <li class="no-toggle">
-                <p class="time">9:00 AM</p>
-                <h3> Welcome &amp; Code of Conduct </h3>
+                <div class="clearfix">
+                  <div class="col-10 col">
+                    <p class="time">9:00 AM</p>
+                    <h3>Welcome &amp; Code of Conduct </h3>
+                  </div>
+                  <div class="col-2 col multiple-speakers">
+                    <p class="speakers">Lateesha Thomas</p>
+                    <p class="speakers">Rebecca Miller-Webster</p>
+                    <ul class="qcSchSpeakers clearfix">
+                      <li>
+                        <a href="#lateesha-thomas" class="tips" title="Lateesha Thomas">
+                          <img src="./assets/img/speakers/lateesha-thomas.jpeg" alt="Lateesha Thomas headshot" />
+                        </a>
+                      </li>
+                      <li>
+                        <a href="#rebecca-miller-webster" class="tips" title="Rebecca Miller-Webster">
```

Comment:
super small, in all the other tooltips we use caps (i think that was carried over from the theme design) so we could make these caps for consistency.

but the tooltips are kind of redundant since we say the names above, so maybe can overlook them in general?